apps/tutorial-servo-can-control/TutorialServo.cpp

```diff
@@ -0,0 +1,45 @@
+
+#include "mbed.h"
+#include "TutorialServo.h"
+
+#include <iostream>
+using namespace std;
+
+
+TutorialServo::TutorialServo(PinName servoPin, float servoRangeInDegrees = 180.0,
```

Comment:
instead of providing default values as you have done we do it by providing something called a member initializer list. Below is a link explaining what that means. Try implementing this in your code.

https://en.cppreference.com/w/cpp/language/constructor